src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -500,8 +500,7 @@ public void receiveSimpleMessage(TestContext context) {
         // consume records
```

Comment:
I would not change every single test in sending with the timeout addition even because we have to test what happens when this param is not specify. I would add a new test for this use case. I know there is a lot of common code in the tests and we have an issue for that in order to factor out common methods.

Reply:
I just changed putting it to the header to putting it into the query. Are you proposing removing timeout from all the tests?